frontend/src/components/DashboardView/index.tsx

```diff
@@ -179,6 +180,22 @@ function DashboardView() {
                   />
                 );
               }
+              if (element.type === 'CHART') {
+                const chartElement = element as DashboardChartConfig;
+                return (
+                  <ChartBlock
+                    // eslint-disable-next-line react/no-array-index-key
+                    key={`chart-block-${index}`}
+                    index={index}
+                    startDate={chartElement.startDate}
+                    endDate={chartElement.endDate}
+                    wmsLayerId={chartElement.wmsLayerId}
+                    adminUnitLevel={chartElement.adminUnitLevel}
+                    adminUnitId={chartElement.adminUnitId}
+                    mode={mode}
+                  />
+                );
+              }
               return <div>Content type not yet supported</div>;
```

Comment:
Can we remove the default return here now that we’re supporting all block types?